lxd/container_lxc.go

```diff
@@ -5973,6 +6015,135 @@ func (c *containerLXC) createNetworkDevice(name string, m types.Device) (string,
 	return dev, nil
 }
 
+func (c *containerLXC) fillVfioNetworkDevice(name string, m types.Device, reserved []string) (types.Device, error) {
+	if m["nictype"] != "vfio" {
+		return m, nil
+	}
```

Comment:
Given we only call this function on vfio devices, we probably should error there.

Reply:
done